### ISSUE TITLE
Presto CLI improvements to validate the LDAP credentials on login

### ISSUE THREAD

**Issue by DRavikanth** [prestosql/presto#4641]
Hi, 

After enabling LDAP auth for Presto, when I pass in options for CLI as follows:

```
root@dataworker101:~ # presto --server testserver.int.blah.com:8080 --catalog hive --schema dwh
presto:dwh> use scratch; show tables;
Error running command: Error starting query at <blah blah blah>
```
I am still able to login, however an error is thrown only after we run a query. Observed the same behavior for incorrect password after LDAP is enabled. It would be nice to validate the credentials at login itself instead of validating after running a query. 

As quoted by @findepi on Slack: 

> this is a consequence of the fact that Presto session state is kept on the client side.
There is no “session” on the server. So, CLI isn’t going to the server on startup.
However, I see this as an unexpected behavior, and not necessarily desirable.
We have same behavior in JDBC, where it could be saving some overhead unwanted
in programmatic usage, but CLI should be as user friendly as possible.
Also, CLI does some background queries to fetch table and function information from
the server.

Thanks,
Ravi

**Comment by tooptoop4** [comment]
duplicate of https://github.com/prestodb/presto/issues/11299
